Rethinking_2/Chp_08.ipynb

```diff
@@ -0,0 +1,1578 @@
+{
```

Comment:
<p>as in a previous cell you may want to replace  tight_layout with constrained_layout </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/resources/pull/88/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>